From 7c44909da69d01b404ebed6fbfb06caff4ebf235 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 18 May 2016 11:39:07 +0200
Subject: [PATCH] location: Update for PermissionStore DBus API changes

As xdg-app has been renamed to Flatpak, the permission store's
bus name and interface name have been changes as well.

https://bugzilla.gnome.org/show_bug.cgi?id=766598
---
 js/ui/status/location.js | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/js/ui/status/location.js b/js/ui/status/location.js
index 0070132..b1e97c7 100644
--- a/js/ui/status/location.js
+++ b/js/ui/status/location.js
@@ -62,8 +62,8 @@ var AgentIface = '<node> \
   </interface> \
 </node>';
 
-var XdgAppIface = '<node> \
-  <interface name="org.freedesktop.XdgApp.PermissionStore"> \
+var PermissionStoreIface = '<node> \
+  <interface name="org.freedesktop.impl.portal.PermissionStore"> \
     <method name="Lookup"> \
       <arg name="table" type="s" direction="in"/> \
       <arg name="id" type="s" direction="in"/> \
@@ -80,7 +80,7 @@ var XdgAppIface = '<node> \
   </interface> \
 </node>';
 
-const PermissionStore = Gio.DBusProxy.makeProxyWrapper(XdgAppIface);
+const PermissionStore = Gio.DBusProxy.makeProxyWrapper(PermissionStoreIface);
 
 const Indicator = new Lang.Class({
     Name: 'LocationIndicator',
@@ -254,8 +254,8 @@ const Indicator = new Lang.Class({
     _connectToPermissionStore: function() {
         this._permStoreProxy = null;
         new PermissionStore(Gio.DBus.session,
-                           'org.freedesktop.XdgApp',
-                           '/org/freedesktop/XdgApp/PermissionStore',
+                           'org.freedesktop.impl.portal.PermissionStore',
+                           '/org/freedesktop/impl/portal/PermissionStore',
                            Lang.bind(this, this._onPermStoreProxyReady));
     },
 
-- 
2.7.4

